### PR TITLE
OAEWDGT-138 - New Hello World tutorial missing content

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.sakaiproject.org/browse/OAEWDGT-138

### CODE REVIEW

File: app/views/sdk/learnbasics.html.erb [hunk 1]
```diff
@@ -114,7 +116,366 @@
...
+
+    <p>If you include a message key in your HTML template that is not defined in a properties file, you will see 'undefined' where the message should appear. If you see 'undefined' in your widget's interface, it's likely you are missing a message key and value pair in your <i>default.properties</i> file.</p>
+
+    <h1>Step 6: <b>Load My Hello World in Sakai OAE</b></h1>
+
+    <p>Now that our widget is configured and the message keys have been defined, we can see My Hello World in Sakai OAE. Of course, since we have not defined our CSS or JavaScript yet, the widget will not do much, but we can test to make sure we have set up our widget properly. Follow these steps to see the My Hello World widget in action.</p>
+
+        <h2>Log into Sakai OAE</h2>
+        <p>Make sure Sakai OAE is running on your machine. If you are unsure about how to do this, look at the <%= link_to "development setup", "/sdk/devsetup", :class => "wl-regular-link" %> page.</p>
```
Comment (apex13) [inline app/views/sdk/learnbasics.html.erb]: @christianv I think we should update this link as a result of https://jira.sakaiproject.org/browse/OAEWDGT-186

File: app/views/sdk/learnbasics.html.erb [hunk 2]
```diff
@@ -114,7 +116,366 @@
...
+
+        <p>Once Sakai OAE is running, you should be able to see the homepage at <%= link_to "http://localhost:8080/", "http://localhost:8080/", :class => "wl-regular-link" %>. If you have not already, create an account by clicking the 'Register here' link on this page.</p>
```
Comment (apex13) [inline app/views/sdk/learnbasics.html.erb]: @christianv The link no longer says "Register Here", but "Sign Up"

File: app/views/sdk/learnbasics.html.erb [hunk 3]
```diff
@@ -114,7 +116,366 @@
...
+});
+</pre>
+    </div>
+
+    <p>Notice that all the code that pertains to this widget is wrapped in a require statement and a <i>sakai_global.helloworld</i> function. This is another measure to prevent clashes with other widgets that may have similarly named variables or functions (i.e. many widgets have a <i>doInit()</i> function). It also provides other parts of the Sakai OAE framework a standard, convenient way to reference this widget's functionality: JavaScript for all widgets is encapsulated in a <i>sakai_global.WIDGET_ID</i> function.</p>
+
+    <p>The overall structure of the sakai_global.myhelloworld function is also important. All widgets should try and follow the same overall structure to make it easier to read and debug widget JavaScript code:</p>
+
+    <ul class="regular_list">
+    <li><strong>Configuration variables</strong> - shared variables this widget's code will need</li>
```
Comment (apex13) [inline app/views/sdk/learnbasics.html.erb]: @christianv The bullet points for these list items are outside of the container.

File: app/views/sdk/learnbasics.html.erb (hunk 1 above)
Comment (christianvuerings) [inline app/views/sdk/learnbasics.html.erb]: @apex13 Thanks for spotting, I also updated other references.

File: app/views/sdk/learnbasics.html.erb (hunk 2 above)
Comment (christianvuerings) [inline app/views/sdk/learnbasics.html.erb]: @apex13 Good catch, updated the naming.

File: app/views/sdk/learnbasics.html.erb (hunk 3 above)
Comment (christianvuerings) [inline app/views/sdk/learnbasics.html.erb]: @apex13 This should now be fine now, as I merged in `master` which should contain the regular_list CSS class. I also changed the indentation of the `<li>`'s

**Pull request merged**